projects/out_of_sample/scripts/experiment-config/prep_oos_experiments.py

```diff
@@ -0,0 +1,168 @@
+
+    prognostic_run_config_template_path = all_experiment_config[
+        "prognostic_run_config_template"
+    ]
+    launch_destination = all_experiment_config["launch_destination"]
+    if launch_destination.startswith("~"):
+        launch_destination = os.path.expanduser(launch_destination)
+    model_config_dir_path = all_experiment_config["model_config_dir_path"]
+
+    segments = all_experiment_config["segments"]
+
+    result_destination_config = all_experiment_config["result_destination"]
+    bucket = result_destination_config["bucket"]
+    project = result_destination_config["project"]
+    experiment_base = result_destination_config["experiment"]
+    trial = result_destination_config["trial"]
+
+    tendency_paths = {
+        "tq": all_experiment_config["base_model"]["tq_tendencies_path"],
+        "wind": all_experiment_config["base_model"]["wind_tendencies_path"],
+    }
+    n = 0
+    run_str = "#!/bin/bash\n\nset -e\n\n"
+    for novelty_detector in all_experiment_config["novelty_detectors"]:
+        nd_name = novelty_detector["nd_name"]
+        nd_path = novelty_detector["nd_path"]
+        for cutoff in novelty_detector["cutoffs"]:
+            n += 1
+            model_config_paths = []
+            for tendency, tendency_path in tendency_paths.items():
+                tendency_id = f"{nd_name}-{cutoff}-{tendency}"
+                model_config_path = os.path.join(model_config_dir_path, tendency_id)
+                temp_config_path = os.path.join(temp_dir.name, tendency_id)
+                model_config_paths.append(model_config_path)
+                write_oos_config(
+                    tendency_path,
+                    nd_path,
+                    cutoff,
+                    model_config_dir_path,
+                    temp_config_path,
+                )
+            prognostic_config_path_dir_name = f"{nd_name}-{cutoff}"
+            prognostic_run_config_dir_path = os.path.join(
+                launch_destination, prognostic_config_path_dir_name
+            )
+            if not os.path.exists(prognostic_run_config_dir_path):
+                os.makedirs(prognostic_run_config_dir_path)
+            prognostic_run_config_path = os.path.join(
+                prognostic_run_config_dir_path, "prognostic-run.yaml"
+            )
+            write_prognostic_run_config(
+                prognostic_run_config_template_path,
+                prognostic_run_config_path,
+                model_config_paths,
+            )
+            experiment = f"{experiment_base}-{nd_name}-{cutoff}"
+            argo_submit_str, job_name, output_path = make_argo_submit_command(
+                bucket,
+                project,
+                experiment,
+                trial,
+                os.path.join(prognostic_config_path_dir_name, "prognostic-run.yaml"),
+                segments,
+            )
+            run_str += argo_submit_str
+            print(f"Experiment #{n}: {nd_name} with cutoff={cutoff}")
```

Comment:
It may be useful to print these lines to file so you have an easily accessible list of the run names and their configurations